index.html

```diff
@@ -1,14 +1,225 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>First Capstone Project</title>
-    <link rel="stylesheet" href="./styles.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quill pen web development</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@700;900&family=Lato&family=Montserrat:wght@700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+
+  <body>
+    <header>
+      <!-- mobile navbar -->
+      <navbar class="mob-navbar">
+        <div>
+          <img class="ham" src="./images/ham.png" alt="" />
+        </div>
+        <a class="logo-link" href="./index.html">
+          <img class="logo" src="./images/personal/quill-pen.png" alt="" />
+        </a>
```

Comment:
Great design @ankitt26 🤝

- [ ] Kindly remove the logo from your mobile screens for both the Home and About pages. This will make your design a replica of the design provided.
- Your design
![mobile-logo](https://user-images.githubusercontent.com/63917725/233445326-9c18a1e9-b647-46e0-a2be-c29030bfb89b.PNG)
- Given design
![given-mobile](https://user-images.githubusercontent.com/63917725/233445387-19f53149-592b-409b-96c6-17ebed592005.PNG)